tinywallet/pyproject.toml

```diff
@@ -34,7 +34,7 @@ pytest-cov = "^2.8.1"
 flake8 = "^3.7.9"
 black = "^19.10b0"
 isort = "^4.3.21"
-Cython = "^0.29.16"
+pytest-qt = "^3.3.0"
```

Comment:
I see `pytest-qt`, but I don't see it used.

Reply:
True, sorry. I started out intending to use it, but then I measured coverage and I noticed that there were improvements to make even before starting to test the UI. Shall I take it out of this PR?

Reply:
Yeah. Please do. *poetry.lock* entry too. Glad to know you're moving this way though.